tests/blackbox.py

```diff
@@ -1,15 +1,18 @@
-from typing import Callable
+from typing import Callable, List, Sequence, Union
```

Comment:
```suggestion
from typing import Callable, Sequence
```

Reply:
@tzaffi Addressed in https://github.com/algorand/pyteal/pull/335/commits/d799e3752e2a92f52102a06ae85f4f499419ae69.  I didn't directly take in the _suggestion_ because I had to make several changes in 1 commit.

If it looks good, can you resolve?

Reply:
👍